### ISSUE TITLE
Installation of influxdb failed on Ubuntu 20.04.2 LTS missing permissions on influxd-systemd-start.sh

### ISSUE THREAD

**Issue by fff0x** [Madic-/ansible-check_mk#1]
Hi Michael, today I tried your Ansible playbook and the influxdb install task, more precisely the Influxdb package for Ubuntu, creates the shellscript `/usr/lib/influxdb/scripts/influxd-systemd-start.sh` that is executed on influxdb startup. But it lacks of the execution permission flag. My workaround was to correct the permissions in `tasks/install_04_influxdb.yml` before *flush_handlers*:

```
- name: InfluxDB | Workaround for Startscript
  file: dest=/usr/lib/influxdb/scripts/influxd-systemd-start.sh mode=a+x
  ignore_errors: yes
```

Afterwards I found the upstream bug report: https://community.influxdata.com/t/influxdb-1-8-7-will-not-start/20965

One additional thing:
As soon as I fixed this step, I run into the next problem with the missing user for the mod-gearman-module and I found your out-commended code. With this enabled, everything runs through.

**Comment by madic-creates** [comment]
Hi fff0x, thank you for the reports.

Regarding the mod-gearman-module: Probably the version from the consol repository got installed. That's not the version I intend to install.

Though I've prepared a fix for both problems and am testing them.

**Comment by madic-creates** [comment]
Fixed in commit 295a3f0
